go-controller/pkg/ovn/libovsdbops/encap.go

```diff
@@ -0,0 +1,34 @@
+package libovsdbops
+
+import (
+	"fmt"
+
+	libovsdbclient "github.com/ovn-org/libovsdb/client"
+
+	"github.com/ovn-org/ovn-kubernetes/go-controller/pkg/sbdb"
+)
+
+// MutateEncapOptions adds the provided options, and fails if the encap table for a given host does not exist in sbdb
+func MutateEncapOptions(sbClient libovsdbclient.Client, chassisName string, options map[string]string) error {
+	encap := &sbdb.Encap{
+		ChassisName: chassisName,
+		Options:     options,
+	}
+
+	opModel := OperationModel{
+		Model: encap,
+		// ChassisName column is not an index, use modelPredicate
+		ModelPredicate: func(en *sbdb.Encap) bool { return en.ChassisName == encap.ChassisName },
+		OnModelMutations: []interface{}{
+			&encap.Options,
+		},
+		ErrNotFound: true,
```

Comment:
A chassis can have multiple encaps:
https://github.com/ovn-org/ovn/blob/187270cc65ba6a1cd3d45c9f27b21bf4a4f8b5ef/ovn-sb.ovsschema#L67
Maybe it would be best to have `BulkOps: true` here?

Reply:
This was removed since it was called by the ovn-kube node code